lib/http/http.c

```diff
@@ -506,6 +507,20 @@ gotheaders(struct http_cookie * H, uint8_t * buf, size_t buflen)
 		H->res.headers[i].header = s;
 		H->res.headers[i].value = s[cpos] ? &s[cpos + 1] : &s[cpos];
 		s[cpos] = '\0';
+
+		/* Strip leading optional whitespace (OWS) from the value. */
+		j = 0;
+		while (H->res.headers[i].value[j] == ' '
+		    || H->res.headers[i].value[j] == '\t')
```

Comment:
Even if you like the idea in general, you might object to this style.  I used `[j]` here so that these lines would be the same as the "strip trailing" case, but they look really weird on their own.